Src/ILGPU.Tests.OpenCL/ILGPU.Tests.OpenCL.csproj

```diff
@@ -66,4 +68,11 @@
     </Compile>
   </ItemGroup>
 
+  <Target Name="TextTemplateTransform">
```

Comment:
It's probably worth consolidating all of these behind a single target file that you can import inside your csproj.